Account/Sources/Account/NewsBlur/Models/NewsBlurFeedChange.swift

```diff
@@ -44,6 +44,10 @@ extension NewsBlurFeedChange: NewsBlurDataConvertible {
 			}
 		}()
 
-		return postData.percentEncodedQuery?.data(using: .utf8)
+		// `+` is a valid character in query component as per RFC 3986 (https://developer.apple.com/documentation/foundation/nsurlcomponents/1407752-queryitems)
+		// workaround:
+		// - http://www.openradar.me/24076063
+		// - https://stackoverflow.com/a/37314144
+		return postData.percentEncodedQuery?.replacingOccurrences(of: "+", with: "%2B").replacingOccurrences(of: "%20", with: "+").data(using: .utf8)
```

Comment:
This may make more sense as an extension to `URLComponents` (in `RSWeb`?) seeing that there's a similar issue for FeedWrangler.